### DISCUSSION TITLE
SEB keeps crashing

### DISCUSSION THREAD

**Discussion by SongHakyung** [SafeExamBrowser/seb-win-refactoring#187] [Q&A]
Hello, I have been encountering this issue since the start of using safe exam browser. I've tried uninstalling and reinstalling SEB, as well as disabling antiviruses but the issues still persists. The pop of message that happens is "SafeExamBrowser has encountered an unrecoverable error! Please consult the log files for more information" or something along the lines of that. Can someone please help me? I've also included the log files from my previous sessions. Hope you guys can help me :D 

[logs.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7114627/logs.zip)s.

**Comment by dbuechel** [comment]
I can't really find a clear pattern, but it appears that sometimes the SEB client can't terminate properly. Could you please elaborate a bit on what exactly happens and when, e.g. does the error message appear when you try to terminate SEB, does SEB terminate at all or does it hang and so on. Also, please let us know which Windows version you're using and whether you have your system up to date (all updates installed etc.).

**Reply by SongHakyung** [reply]
Hello, thank you so much for the reply. The message appears in the middle of the session, but if I dont click okay, I can still hang on for a but longer, however it will eventually shut down on its own. My system is windows 10 and it is currently up to date.

**Reply by dbuechel** [reply]
I have seen some error messages related to WCF, which is being used for communication between the different application parts of SEB. I can't really see a specific issue, it might help to update / repair the .NET Framework installation on your machine.
